checkov/common/bridgecrew/integration_features/features/suppressions_integration.py

```diff
@@ -160,6 +160,12 @@ def _check_suppression(self, record: Record, suppression: dict[str, Any]) -> boo
                         return True
             return False
 
+        elif type == 'LicenseType':
+            for license_type in suppression['licenseTypes']:
```

Comment:
I see in the prev types ('Cves' 'CvesAccounts') there are check for `if self.bc_integration.repo_id in suppression['accountIds']`
are you sure we don't need it in our case?

Reply:
cool